persistence-jdbc/scaladsl/src/multi-jvm/scala/com/lightbend/lagom/javadsl/persistence/jdbc/JdbcClusteredPersistentEntitySpec.scala

```diff
@@ -17,15 +21,15 @@ object JdbcClusteredPersistentEntityConfig extends AbstractClusteredPersistentEn
       lagom.serialization.play-json {
         serialization-registry = "com.lightbend.lagom.scaladsl.persistence.TestEntitySerializerRegistry"
       }
-    """
-  )
+    """)
 }
 
 class JdbcClusteredPersistentEntitySpecMultiJvmNode1 extends JdbcClusteredPersistentEntitySpec
 class JdbcClusteredPersistentEntitySpecMultiJvmNode2 extends JdbcClusteredPersistentEntitySpec
 class JdbcClusteredPersistentEntitySpecMultiJvmNode3 extends JdbcClusteredPersistentEntitySpec
 
-class JdbcClusteredPersistentEntitySpec extends AbstractClusteredPersistentEntitySpec(JdbcClusteredPersistentEntityConfig) {
+class JdbcClusteredPersistentEntitySpec extends AbstractClusteredPersistentEntitySpec(JdbcClusteredPersistentEntityConfig)
+  with JdbcPersistenceComponents with DBComponents with HikariCPComponents {
```

Comment:
let me know if there is a more idiomatic way to structure this test (and its base AbstractClusteredPersistentEntitySpec)